src/controllers/features/limited/isThisYourCompanyController.ts

```diff
@@ -7,11 +7,12 @@ import { COMPANY_DETAILS, SUBMISSION_ID, USER_DATA, GET_ACSP_REGISTRATION_DETAIL
 import { saveDataInSession } from "../../../common/__utils/sessionHelper";
 import { getAcspRegistration } from "../../../services/acspRegistrationService";
 import logger from "../../../utils/logger";
-import { AcspData, Company } from "@companieshouse/api-sdk-node/dist/services/acsp";
+import { AcspData } from "@companieshouse/api-sdk-node/dist/services/acsp";
 import { ErrorService } from "../../../services/errorService";
 import { CompanyDetailsService } from "../../../services/company-details/companyDetailsService";
 import { isThisYourCompanyAnswers } from "../../../services/checkYourAnswersService";
 import { AcspDataService } from "../../../services/acspDataService";
+import { Company } from "../../../model/Company";
```

Comment:
We would still want to use the Company from the api-sdk-node

Reply:
Not sure why this was marked resolved. I think the Company type should be used from data model coming from api-sdk-node

Reply:
Company type from api-sdk-node does only contains companyName and companyNumber. Here it is the company type used to show the other values on is this your company screen which contains status etc